web/src/StoreEditPage.js

```diff
@@ -313,6 +313,20 @@ class StoreEditPage extends React.Component {
             }} />
           </Col>
         </Row>
+        <Row style={{marginTop: "20px"}} >
+          <Col style={{marginTop: "5px"}} span={(Setting.isMobile()) ? 22 : 2}>
+            {i18next.t("store:SuggestionsCount")}:

```

Comment:
SuggestionsCount -> Suggestion count

Just copy my word if you cannot make it right